src/server/rdb_load.cc

```diff
@@ -1814,6 +1827,32 @@ error_code RdbLoaderBase::HandleCompressedBlobFinish() {
   return kOk;
 }
 
+error_code RdbLoaderBase::HandleJournalEntries(Service* service) {
+  size_t len;
+  bool _encoded;
+  SET_OR_RETURN(LoadLen(&_encoded), len);
+
+  // TODO: Do this inline into buffer
+  string res;
+  SET_OR_RETURN(FetchGenericString(), res);
```

Comment:
It would be better if we read into the internal buffer and then used a `BytesSource` just to reference the segment there